en/user_guide/internal_link.rst

```diff
@@ -0,0 +1,20 @@
+Internal Link
+=============
+
+Open Orchestra use TinyMce to editing rich text.
+A custom button allows to add an internal link, ie a link to a page managed by the Back Office.
```

Comment:
``a custom button allows`` is weird in this context, I'd rather say ``a custom button is used to`` here